src/resources/filters/crossref/equations.lua

```diff
@@ -46,7 +46,17 @@ function process_equations(blockEl)
         if _quarto.format.isLatexOutput() then
           targetInlines:insert(pandoc.RawInline("latex", "\\begin{equation}"))
           targetInlines:insert(pandoc.Span(pandoc.RawInline("latex", eq.text), pandoc.Attr(label)))
-          targetInlines:insert(pandoc.RawInline("latex", "\\label{" .. label .. "}\\end{equation}"))
+
+          -- Pandoc 3.1.7 started outputting a shadow section with a label as a link target
```

Comment:
Since we're bundling Pandoc ourselves and we tell everyone that they should always just use the correct Pandoc version, should we even be adding this? The only other place we use pandoc.types.Version is in `layout/table.lua:155` and I'm pretty sure we should just drop that entirely...

Reply:
Duh yes you're completely correct. I'll update.